### PR TITLE
[FEAT] React Router Dom Added and Test API Call

### PR DESCRIPTION
**Pull request by yf-1059**

### CODE REVIEW

File: src/containers/Homepage.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Header from '../components/Header';
+import Sidebar from '../components/Sidebar.js';
+import BookingPage from '../components/BookingPage.js';
+import '../sass/Homepage.scss';
```
Comment (shihao92) [inline src/containers/Homepage.js]: usually to save the stylesheets we will be saving into the folder stylesheets and if the stylesheet is related with the containers or components we will be putting the scss inside the particular folder and import them here

Comment (shihao92) [reply src/containers/Homepage.js]: so can you rename the sass folder into stylesheets

File: src/containers/TestAPICall/Index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <div className='container mt-4'>
+        <div className="row justify-content-center title fw-normal">
+          Test API Call
+        </div>
+      </div>
+      <div className="container mt-4">
+        <table className='table table-bordered table-dark'>
+          <thead>
+            <tr className='text-center'>
+              <th scope="col">ID</th>
```
Comment (shihao92) [inline src/containers/TestAPICall/Index.js]: why do you need the scope props here?

**Review by shihao92 — CHANGES_REQUESTED** [review]

**Review by shihao92 — APPROVED** [review]

**Comment by shihao92** [comment]
usually our partial scss file will be using small letter

**Comment by yf-1059** [comment]
> usually our partial scss file will be using small letter

Alright, noted. Will change it in next pull request.

**Pull request merged**